admin-ui/src/components/Widget/ReviewDetailWidget/ReviewDetailWidgetConfigSetting.tsx

```diff
@@ -41,6 +41,14 @@ const RatingWidgetConfigSetting = ({name}: any) => {
                                 })
                             }}/>
                         </SidebarDetailField>
+                        <SidebarDetailField>
+                            <Label className={"frt-text-xs"} htmlFor="none">Reply background Color</Label>
```

Comment:
Why the lable is Reply Background color, and the property is named as secondary color. updae the label to Reply background color